src/otg/dataset/summary_statistics.py

```diff
@@ -175,7 +102,9 @@ def from_gwas_harmonized_summary_stats(
             _df=processed_sumstats_df,
         )
 
-    def calculate_confidence_interval(self: SummaryStatistics) -> SummaryStatistics:
+    def calculate_confidence_interval_for_summary_statistics(
```

Comment:
This function is not used at the moment.
If possible, I think it is a good idea if the SummaryStatistics dataset reports the diff confidence interval bounds, and drops `standardError` from the schema (unless it is used downstream).

Reply:
I dropped this function. The logic to generate the confidence intervals for the betas is quite source specific, so I think there's no need to have a shared function.

Reply:
Thanks!